include/mitsuba/render/mesh.h

```diff
@@ -24,6 +25,7 @@ class MTS_EXPORT_RENDER Mesh : public Shape<Float, Spectrum> {
     using typename Base::ScalarSize;
     using typename Base::ScalarIndex;
 
+    using ScalarIndex3 = std::array<ScalarIndex, 3>;
```

Comment:
Maybe could move this alias to the constructor itself?
Also use `Array` (enoki) instead of `std::array` for consistency. (minor)